helm_deploy/values-ips.yml

```diff
@@ -0,0 +1,49 @@
+whitelist_groups:
+
+  internal:
+    health-kick: 35.177.252.195/32
+    mojvpn: 81.134.202.29/32
+    office: 217.33.148.210/32
+    petty_france_wifi: 213.121.161.112/28
+  
+  cloudplatform:
+    cloudplatform-live1-1: 35.178.209.113/32
+    cloudplatform-live1-2: 3.8.51.207/32
+    cloudplatform-live1-3: 35.177.252.54/32
+
+  ark:
+    ark-dom1-corsham-psn: 51.247.4.0/24
+    ark-dom1-corsham: 194.33.196.0/24
+    ark-dom1-farnborough-psn: 51.247.3.0/24
+    ark-dom1-farnborough: 194.33.192.0/24
+    ark-dom1-non-live-1: 194.33.193.0/25
+    ark-dom1-non-live-2: 194.33.197.0/25
+    ark-dom1-ttp1: 195.59.75.0/24
+    ark-nps-hmcts-ttp2: 194.33.192.0/25
+    ark-nps-hmcts-ttp4: 194.33.196.0/25
+
+  prisons:
+    dxc-mitcheldean: 195.92.38.16/28
+    quantum1: 62.25.109.197/32
+    quantum2: 212.137.36.230/32
+    digitalprisons1: 52.56.112.98/32
+    digitalprisons2: 52.56.118.154/32
+    j5-phones-1: 35.177.125.252/32
+    j5-phones-2: 35.177.137.160/32
+    thameside-private-prison: 217.22.14.151/32
+    oakwood: 194.176.200.113/32
+
+  externalProbationAgencies:
+    interservfls: 51.179.196.131/32
+    sodexo-northumberland: 88.98.48.10/32
+    sodexo1: 80.86.46.16/32
+    sodexo2: 80.86.46.17/32
+    sodexo3: 80.86.46.18/32
+
+  pentesting:
+    pentester-bsi-1: 145.239.254.21/32
+    pentester-bsi-2: 54.37.241.156/30
+ 
+  other: 
+    durham-tees-valley: 51.179.193.241/32
```

Comment:
this file is arbitrarily structured. We've removed the pentesting whitelisting from cat tool since, so it's a bit out of date